.github/workflows/run_tests.yml

```diff
@@ -49,3 +42,9 @@ jobs:
         npm install puppeteer
         export ORD_EDITOR_POSTGRES_PASSWORD=test
         ./run_tests.sh "${CACHE_TARGET}"
+    - name: Update docker cache
+      run: |
+        cd "${GITHUB_WORKSPACE}"
+        docker tag openreactiondatabase/ord-editor "${CACHE_TARGET}" && docker push "${CACHE_TARGET}" || true
```

Comment:
"|| true"?

Reply:
The test shouldn't fail if the cache can't be updated for some reason. But maybe we want the failure so we can spot caching errors. I'll remove.